### ISSUE TITLE
Step 1 Preprocessing notebook doesn't work

### ISSUE THREAD

**Issue by ksolarski** [jrchang612/MoonBoardRNN#3]
First of all, amazing project; as a climber myself I enjoyed reading your paper. I actually haven't tried MoonBoard yet but I would assume that some holds are much better than another ones. One could deduce which holds are good: if one hold is primarily used in very difficult boulders, this would imply that it's not a good hold. Is information like this incorporated in the Neural Network architecture?

I also found some issues with running the code you posted. The notebook attached below doesn't work as there are some undefined parameters in the code. Most of those bugs are very easy to fix:
https://github.com/jrchang612/MoonBoardRNN/blob/master/preprocessing/Step1_data_preprocessing_v2.ipynb

Step 3 Notebook also doesn't work with newer Tensorflow versions as it's impossible to use the custom loss function that corresponds to two layers the way you did. Could you add which package versions you used in your project?

**Comment by Beakerboy** [comment]
The Fork at https://github.com/BouldAR-Warwick/MoonBoardRNN seems to have some improvements, including one called "Constants"
